src/2_add_cots_priority.jl

```diff
@@ -0,0 +1,51 @@
+using
+    CSV,
+    Dates,
+    DataFrames
+
+using
+    GLMakie,
+    GeoMakie
+
+using
+    Statistics,
+    Bootstrap
+
+import GeoDataFrames as GDF
+import GeoFormatTypes as GFT
+import ArchGDAL as AG
+
+include("common.jl")
+
+#Load required data
+RRAP_lookup = GDF.read(joinpath(OUTPUT_DIR, "rrap_shared_lookup.gpkg"))
+cots_priority = CSV.read(joinpath(DATA_DIR, "CoCoNet_CoTS_control_reefs_2024.csv"), DataFrame, missingstring="NA")
+
+#Format data
+rename!(cots_priority, Dict("LAT" =>  :cots_LAT, "LON" => :cots_LON,
+    "ReefID" => :GBRMPA_ID, "Priority" => :priority))
+cots_priority = cots_priority[:,[:GBRMPA_ID,:cots_LON,:cots_LAT,:priority]]
+
+#Adding cots priority into RRAP_lookup
+RRAP_lookup = leftjoin(RRAP_lookup, cots_priority, on=:GBRMPA_ID, order = :left)
+println(names(RRAP_lookup))
+
+#cross-checking that the GBRMPA_ID from RRAP_lookup matches from cots_priority (within 100m - rounded values)
```

Comment:
Prefer sentence casing for comments that are not inline:

e.g., capitalize first character of first word.



```
# This is preferred:
# Cross-checking that the ...

x = 100;  # inline comments (those that are on the same line as code) do not have to follow sentence case
```